input/elasticapm/internal/modeldecoder/generator/slice.go

```diff
@@ -87,6 +87,7 @@ for _, elem := range val.%s{
 func sliceRuleMinVals(w io.Writer, f structField, rule validationRule) error {
 	fmt.Fprintf(w, `
 for _, elem := range val.%s{
+	//lint:ignore SA4003 don't reject the check for uint
```

Comment:
This is fine for now, but I think we should probably update the generators to remove the redundant check from the generated Go code. To do that I think we would remove the `minVals` tag from the unsigned int field, and infer a minimum value in the generated JSON Schema.

Could you please open an issue to track this?

Reply:
See https://github.com/elastic/apm-data/issues/131